rotkehlchen/accounting/transactions.py

```diff
@@ -108,15 +109,17 @@ def process(
             taxable=event_settings.taxable,
             count_entire_amount_spend=event_settings.count_entire_amount_spend,
             count_cost_basis_pnl=event_settings.count_cost_basis_pnl,
-            extra_data={'tx_hash': event.tx_hash.hex()},
+            extra_data={
+                'tx_hash': event.tx_hash.hex() if isinstance(event, EvmEvent) else event.event_identifier,  # TODO: undo. Is a temporary hack  # noqa: E501
```

Comment:
Same with the todo. Unless you mark it in a special way will be hard for you to find them later